lib/terminology_table.erb

```diff
@@ -35,9 +35,15 @@
                   <li>
                     <% #TODO: Adjust displays to be consistent once all resource differentiate system from info url %>
                     <% if terminology['info_link'] %>
-
                       <%= "<p>Details: <a target=\"_blank\" href=\"#{terminology['info_link']}\">#{terminology['display']}</a><br/>" %>
-                      <%= "System: #{terminology['system']} </p>" %>
+                      <% if terminology['system'].kind_of?(Array) %>
+                        <% for value in terminology['system'] %>
+                          <%= "System: #{value} <br/>" %>
+                        <% end %>
+                      <% else %>
+                        <%= "System: #{terminology['system']}" %>
+                      <% end %>
```

Comment:
instead of checking kind_of? let's just always wrap `terminology['system']` in an array

replace L39 - L45 with:
```
<% for value in Array(terminology['system']) %>
  <%= "System: #{value} <br/>" %>
<% end %>
```

Reply:
Tried this out. It works for me as well

Reply:
changed [here](https://github.com/cerner/fhir.cerner.com/compare/related-person-updates#diff-8b88b19294440cc71880602394d6fbcfR39)